dist/common/scripts/scylla-housekeeping

```diff
@@ -86,9 +92,9 @@ def version_compare(a, b):
     return parse_version(a) < parse_version(b)
 
 
-def create_uuid_file(fl):
-    with open(args.uuid_file, 'w') as myfile:
-        myfile.write(str(uuid.uuid1()) + "\n")
+def create_uuid_file(uuid_filename):
+    with open(uuid_filename, 'w') as uuid_file:
+        uuid_file.write(str(uuid.uuid1()) + "\n")
 
```

Comment:
I'm really confused, I have no idea what this patch does.

Reply:
In this case args from the global scope was used instead the parameter of the file name passed to this function